plugins/networkdelay/parameters.go

```diff
@@ -0,0 +1,18 @@
+package networkdelay
+
+import (
+	"github.com/iotaledger/hive.go/configuration"
+)
+
+// ParametersDefinition contains the definition of the parameters used by the networkdelay plugin.
+type ParametersDefinition struct {
+	// OriginPublicKey defines the default issuer node public key in base58 encoding.
+	OriginPublicKey string `default:"9DB3j9cWYSuEEtkvanrzqkzCQMdH1FGv3TawJdVbDxkd" usage:"default issuer node public key"`
+}
+
+// Parameters contains the configuration used by the networkdelay plugin.
+var Parameters = ParametersDefinition{}
+
+func init() {
+	configuration.BindParameters(&Parameters, "networkdelay")
```

Comment:
In https://goshimmer.docs.iota.org/implementation_design/configuration_parameters.html we state that the optionalNamespace is lowerCamelCase but in the code we use lowereverything. Which one is the correct one?

func init() {
    configuration.BindParameters(&Parameters, "customPlugin")
}

Reply:
This should also be part of https://github.com/iotaledger/goshimmer/issues/1507.